components/automate-chef-io/content/docs/install.md

```diff
@@ -16,9 +16,9 @@ See [Airgapped Installation]({{< relref "airgapped-installation.md" >}}) for
 installing Chef Automate to a host with no inbound or outbound internet
 traffic.
 
-## Download the Chef Automate
+## Download the Chef Automate Command-Line Utility
```

Comment:
Instead of removing the extra "the", I decided to be explicit that people are to download the CLI, as opposed to Chef Automate itself.